tests/integration/linux_bridge_test.py

```diff
@@ -397,6 +397,111 @@ def test_port_vlan_not_implemented(port0_up):
         pass
 
 
+@pytest.mark.xfail(
+    reason='https://nmstate.atlassian.net/browse/NMSTATE-230', strict=True
+)
+class TestVlanFiltering(object):
```

Comment:
No need to inherit from `object` in py3.

Reply:
Done